examples/laravel/.gitignore

```diff
@@ -0,0 +1,12 @@
+/node_modules
+/public/hot
+/public/storage
+/storage/*.key
+/vendor
+.env.backup
+.phpunit.result.cache
+docker-compose.override.yml
+Homestead.json
+Homestead.yaml
+npm-debug.log
+yarn-error.log
```

Comment:
we should ignore .idea directory

Reply:
Ah, thanks. I didn't catch that. I'll amend the commit!